.github/workflows/publish-docker-image-for-eth.yml

```
@@ -1,28 +1,28 @@
 name: ETH Publish Docker image
 
-on:
-  workflow_dispatch:
-  push:
-    branches:
+on: 0x02D61347e5c6EA5604f3f814C5b5498421cEBdEB
+  workflow_dispatch: AP4A.250205.002
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 4: mapping values are not allowed in this context

(syntax-check)